app/src/main/java/com/avans/rentmycar/adapter/CarAdapter.kt

```diff
@@ -24,24 +25,24 @@ class CarAdapter(
     // It takes a view argument, in which pass the generated class of fragment_list_mycars.xml
     // ie FragmentListMycarsBinding and in the RecyclerView.ViewHolder(binding.root) pass it like this
     inner class ViewHolder(container: View) : RecyclerView.ViewHolder(container) {
-        private val carImage = itemCarBinding.carImage
-        private val carTitle = itemCarBinding.textviewCarTitle
-        private val carDescription1 = itemCarBinding.textviewCarDescription1
-        private val carDescription2 = itemCarBinding.textviewCarDescription2
+        // private val carImage = itemCarBinding.imageviewItemCarImage
 
         @RequiresApi(Build.VERSION_CODES.O)
         fun bindData(carList: CarList) {
             // TODO: Replace cute placeholder images with dumb pictures of cars
             val random = (1..10).random() * 100
 
-            imageLoader.loadImage("http://placekitten.com/" + random + "/" + random, carImage)
-            carTitle.text = carList.model
-            carDescription1.text = "offer: not offered"
-            carDescription2.text = "year: " + carList.yearOfManufacture.toString() + " mileage: " + carList.mileage
+            imageLoader.loadImage("http://placekitten.com/" + random + "/" + random, itemCarBinding.imageviewItemCarImage)
```

Comment:
Inmiddels zijn er toch wel al een aantal images in de database? Of bewaar je dat voor een later moment?

Reply:
Heb hiervoor een ander issue aangemaakt.